dev/NavigationView/NavigationView.xaml

```diff
@@ -506,10 +525,10 @@
                                         <!-- "Non header" content -->
                                         <Grid x:Name="ItemsContainerGrid" Grid.Row="6" Margin="0,0,0,8">
                                             <Grid.RowDefinitions>
-                                                <RowDefinition Height="Auto" MinHeight="{ThemeResource NavigationViewItemOnLeftMinHeight}"/> <!-- MenuItems -->
+                                                <RowDefinition x:Name="MenuItemsRow" Height="Auto"/> <!-- MenuItems -->
                                                 <RowDefinition Height="*" MinHeight="21"/> <!-- Fill spacing -->
```

Comment:
I think we need to remove these 21 px from here as well. If there's no footer items and no pane footer - main menu items should have all the space given.